evaluator/src/main/kotlin/PackageRule.kt

```diff
@@ -136,7 +136,8 @@ open class PackageRule(
      */
     fun licenseRule(name: String, licenseView: LicenseView, block: LicenseRule.() -> Unit) {
         resolvedLicenseInfo.filter(licenseView, filterSources = true)
-            .applyChoices(ruleSet.ortResult.getLicenseChoices(pkg.id)).forEach { resolvedLicense ->
+            .applyChoices(ruleSet.ortResult.getLicenseChoices(pkg.id))
+            .applyChoices(ruleSet.ortResult.getRepositoryLicenseChoices()).forEach { resolvedLicense ->
```

Comment:
Shouldn't the repository license choices get priority? For example, if there is a global choice for a package to chose "a" from "a OR b", but in my specific project I cannot use license "a" for some reason and want to chose "b" instead, there is currently no way to do this, right?
I think the problem is we have to axes of specificity here, one is being specific to a package, the other is being specific to the scope of a project.
One way to solve this would be to allow package specific choices in .ort.yml as well, they would then get highest priority. Is this a use case you are interested in?
@sschuberth @tsteenbe Your opinion would be interesting here as well.

Reply:
I think there might be a confusion here. The package specific license choices are already contained in the .ort.yml. There are no global package license choices as of now. The license choices applied in the marked code are the choices of the repository specific package choices and, as you proposed, take precedence to the repository license choices.

Reply:
Yes, that's right, I indeed confused that.